### DISCUSSION TITLE
Tigo CCA + TAP

### DISCUSSION THREAD

**Discussion by Lagerkugel** [mletenay/home-assistant-tigo#8] [Q&A]
Hello friends,
I have patched my CCA module and now have access to the WEB interface. It is colorful, but not structured.
I can see that my TAP module is recognized. But how do I integrate my TSA-A-O optimizers?
I have not found an input field in which I can enter the serial numbers of the optimizers.
I would be grateful for your help.
Best wishes from Germany

**Comment by nblount** [comment]
I think you have to load the optimizers via the Tigo Cloud: https://ei.tigoenergy.com
You goto layout and upload the serial numbers of the optimizers and the panel they are attached to.
